mobilizon_bots/main.py

```diff
@@ -16,11 +22,27 @@ async def main():
     :return:
     """
     logging.config.dictConfig(settings.logging)
+    active_publishers = get_active_publishers(settings)
+
     db = MobilizonBotsDB(Path(settings.db_path))
     await db.setup()
-    published_events = get_published_events()
+
+    # Load past events
+    published_events = list(await get_published_events())
+
+    # Pull unpublished events from Mobilizon
     unpublished_events = get_unpublished_events(published_events)
-    event = select_event_to_publish()
+    # Store in the DB only the ones we din't know about
+    await create_unpublished_events(unpublished_events, active_publishers)
+    unpublished_events = list(await get_db_unpublished_events())
```

Comment:
questa parte è toccata anche in altre PR. mergiamo la tua per ultima e riconciliamo tutto